packages/m2m-gateway/src/services/eserviceService.ts

```diff
@@ -197,6 +198,76 @@ export function eserviceServiceBuilder(
     )({
       condition: isPolledVersionAtLeastResponseVersion(response),
     });
+  // eslint-disable-next-line max-params
+  async function assignEServiceDescriptorAttributesGroup(
+    eserviceId: EServiceId,
+    descriptorId: DescriptorId,
+    groupIndex: number,
+    attributesIds: string[],
+    attributeKind: keyof catalogApi.Attributes,
+    headers: M2MGatewayAppContext["headers"]
+  ): Promise<attributeRegistryApi.Attribute[]> {
+    const eservice = await retrieveEServiceById(headers, eserviceId);
+    const descriptor = retrieveEServiceDescriptorById(eservice, descriptorId);
+    const kindAttributeGroups = descriptor.attributes[attributeKind];
+    if (kindAttributeGroups.length <= groupIndex) {
+      throw eserviceDescriptorAttributeGroupNotFound(descriptorId);
+    }
+    const updateAttributeGroups = [
+      ...kindAttributeGroups[groupIndex],
+      ...attributesIds.map((id) => ({
+        id,
+        explicitAttributeVerification: false,
+      })),
+    ];
+    const updateAttributesSeed = {
```

Comment:
Also here, align the logic to the delete, defining a variable called updatedGroups, then use it to update the seed: https://vscode.dev/github/pagopa/interop-be-monorepo/blob/release/2.8.0/packages/m2m-gateway/src/services/eserviceService.ts#L209